common/src/main/resources/docker/docker.bom

```diff
@@ -509,8 +513,8 @@ brooklyn.catalog:
           }
           rm -f "${PID_FILE}" # docker won't overwrite
           DOCKER="docker run -d --cidfile \"${PID_FILE}\" --net=host"
-          [ "${DOCKER_RUN_VOLUMES}" ] && DOCKER="${DOCKER} $( parse_docker_volumes )"
-          [ "${DOCKER_RUN_ENV}" ] && DOCKER="${DOCKER} $( parse_docker_env )"
+          [[ $DOCKER_RUN_VOLUMES != [] ]] && DOCKER="${DOCKER} $( parse_docker_volumes )"
+          [[ $DOCKER_RUN_ENV != {} ]] && DOCKER="${DOCKER} $( parse_docker_env )"
```

Comment:
I don't believe so, these are set from a DSL brooklyn:config call.
There are tests which cover the config being empty and the config being full- they both pass with this change.